github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```
@@ -113,11 +157,12 @@ periodics:
     repo: project-infra
     base_ref: main
   labels:
-    preset-docker-mirror: "true"
+    preset-docker-mirror-proxy: "true"
     preset-github-credentials: "true"
     preset-podman-in-container-enabled: "true"
     preset-kubevirtci-quay-credential: "true"
-  cluster: kubevirt-prow-workloads
+    preset-gcs-credentials: "true"
+  cluster: s390x-prow-virtual
```

Should this be updated to the bare-metal cluster? 
```suggestion
  cluster: prow-s390x-workloads
```